pom.xml

```diff
@@ -1343,6 +1355,14 @@
             </systemPropertyVariables>
             <trimStackTrace>false</trimStackTrace>
             <reportFormat>plain</reportFormat>
+            <argLine>
+              ${argLine}
```

Comment:
I spend several days trying to be able to add the --add-opens we need without making some esoteric maven plugin or profile to fail. This is the solution I found and it seems to be the official one. What this does is to expand the argLine property we set in our different profiles and then add the --add-opens we need in Java > 16